data/human/transport missions.txt

```diff
@@ -3582,3 +3582,714 @@ mission "Hauler VI cargo prototype 3"
+
+
+
+
+mission "Journalism 1"
+	name "Transport exposition to <planet>"
+	description `Transport expo equipment to <destination>.`
+	minor
+	source
+		attributes "paradise"
+		not attributes "mining"
+		not attributes "textiles"
+		not attributes "research"
+		not attributes "factory"
+	destination "Moonshake"
+	cargo "exposition installations" 5
+	to offer
+		random < 80
+		has "Expedition to Hope 2: done"
+		has "FW Katya 4: done"
+
+	on offer
+		conversation
+			`As you explore the luxurious spaceport you notice a large group of people around one of the larger hangers.`
+			choice
+				`	(Check it out.)`
+				`	(Not right now.)`
+					defer
+			`	You approach some of the people at the end of the line and ask what's going on. "It's the last day of Adelita's exposition on the evacuation of Hope," says an excited young woman. "It was their first work that really got them famous. The last week of their expositions are always free. This is a great chance to see it if you haven't already."`
+			`	The line seems to be moving pretty quickly.`
+			choice
+				`	"Sure, why not."`
+				` 	"Maybe some other time."`
+					defer
+			`	"You're going to love it," she says. "No one knows who they really are but Adelita's work is what inspired me to study journalism. Their mixture of mediums, their eye for story telling... I just love it. I have seen a lot of their stuff but never this expo on Hope. I'm so excited!"`
+			`	You reach the entrance to the "expo" soon enough. Inside is a large room filled with photos, videos, interviews and other mixed media. They focus on three groups: the refugees of Hope, the Navy that saved them, and the Paradise planets that protested the evacuation. There is an installation with nearly a hundred original protest signs from rallies against the Dirt Belt and Navy. Some of them are quite crude and tasteless. Since you are on a Paradise planet it's a strong rebuke.`
+			` There are many, many photos but one especially catches your eye. It's of a woman standing in line to board a Navy destroyer. She is holding a baby at her hip and staring at the camera with a look of sad determination. Their cheeks, rosy and chapped, stand out in contrast with the cold blues in the background. It gives you goosebumps.`
+			`	While good, you were hoping for a bit more as you head for the exit. As the automatic doors open you are hit with an unexpected arctic blast. Instead of back in the spaceport you find yourself in a life sized 3D recording of the evacuation of Hope, complete with refrigeration. The projection gives the illusion of a horizon. There are huge Navy destroyers slowly collecting ash in a snowy field near a small town as hundreds of refugees slowly board. But what stands out most is the giant volcano erupting in the background.`
```

Comment:
```suggestion
			`	While good, you were hoping for a bit more as you head for the exit. As the automatic doors open you are hit with an unexpected arctic blast. Instead of back in the spaceport you find yourself in a life sized 3D recording of the evacuation of Hope, complete with refrigeration. The projection gives the illusion of a horizon. There are huge Navy Cruisers slowly collecting ash in a snowy field near a small town as hundreds of refugees slowly board. But what stands out most is the giant volcano erupting in the background.`
```